docs/examples/opentelemetry-example-app/src/opentelemetry_example_app/flask_example.py

```diff
@@ -29,13 +29,16 @@
 )
 
 trace.set_tracer_provider(TracerProvider())
```

Comment:
Nit: Add the comment about setting tracer_provider before instrumenting here too?

Reply:
Done.